oxrdflib/__init__.py

```diff
@@ -1,274 +1,9 @@
-    if term == DATASET_DEFAULT_GRAPH_ID:
-        return ox.DefaultGraph()
-    if isinstance(term, URIRef):
-        return ox.NamedNode(term)
-    if isinstance(term, BNode):
-        return ox.BlankNode(term)
-    if isinstance(term, Literal):
-        return ox.Literal(term, language=term.language, datatype=ox.NamedNode(term.datatype) if term.datatype else None)
-    if isinstance(term, Graph):
-        return _to_ox(term.identifier)
-    if isinstance(term, tuple):
-        if len(term) == 3:
-            return ox.Quad(_to_ox(term[0]), _to_ox(term[1]), _to_ox(term[2]), _to_ox(context))
-        if len(term) == 4:
-            return ox.Quad(_to_ox(term[0]), _to_ox(term[1]), _to_ox(term[2]), _to_ox(term[3]))
-    raise ValueError(f"Unexpected rdflib term: {term!r}")
-
-
-def _to_ox_quad_pattern(triple: _TriplePattern, context: Optional[Graph] = None):
-    (s, p, o) = triple
-    return _to_ox_term_pattern(s), _to_ox_term_pattern(p), _to_ox_term_pattern(o), _to_ox_term_pattern(context)
-
-
-def _to_ox_term_pattern(
-    term: Optional[Union[URIRef, BNode, Literal, Graph]],
-) -> Optional[Union[ox.NamedNode, ox.BlankNode, ox.Literal]]:
-    if term is None:
-        return None
-    if isinstance(term, URIRef):
-        return ox.NamedNode(term)
-    if isinstance(term, BNode):
-        return ox.BlankNode(term)
-    if isinstance(term, Literal):
-        return ox.Literal(term, language=term.language, datatype=ox.NamedNode(term.datatype) if term.datatype else None)
-    if isinstance(term, Graph):
-        return _to_ox(term.identifier)
-    raise ValueError(f"Unexpected rdflib term: {term!r}")
-
-
-def _from_ox_graph_name(graph_name: Union[ox.NamedNode, ox.BlankNode, ox.DefaultGraph], store: OxigraphStore) -> Graph:
-    if isinstance(graph_name, ox.NamedNode):
-        return Graph(identifier=URIRef(graph_name.value), store=store)
-    if isinstance(graph_name, ox.BlankNode):
-        return Graph(identifier=BNode(graph_name.value), store=store)
-    if isinstance(graph_name, ox.DefaultGraph):
-        return Graph(identifier=DATASET_DEFAULT_GRAPH_ID, store=store)
-    raise ValueError(f"Unexpected Oxigraph graph name: {graph_name!r}")
-
-
-def _from_ox(
-    term: Optional[Union[ox.NamedNode, ox.BlankNode, ox.Literal, ox.Triple]],
-) -> Optional[Union[Node, Tuple[Node, Node, Node]]]:
-    if term is None:
-        return None
-    if isinstance(term, ox.NamedNode):
-        return URIRef(term.value)
-    if isinstance(term, ox.BlankNode):
-        return BNode(term.value)
-    if isinstance(term, ox.Literal):
-        if term.language:
-            return Literal(term.value, lang=term.language)
-        return Literal(term.value, datatype=URIRef(term.datatype.value))
-    if isinstance(term, ox.Triple):
-        return _from_ox(term.subject), _from_ox(term.predicate), _from_ox(term.object)
-    raise ValueError(f"Unexpected Oxigraph term: {term!r}")
+from .parser import OxigraphNTriplesParser, OxigraphRdfXmlParser, OxigraphTurtleParser
```

Comment:
removing public access to `parser` and `store` , only access to specific classes

Reply:
thank you!